cmd/conduit/main.go

```diff
@@ -50,6 +50,11 @@ func init() {
 
 // runConduitCmdWithConfig run the main logic with a supplied conduit config
 func runConduitCmdWithConfig(cfg *conduit.Config) error {
+	defer func() {
+		if r := recover(); r != nil {
+			logger.Fatalf("conduit framework experienced a panic: %v", r)
+		}
+	}()
```

Comment:
Do you think it's worth making a helper function? I'm thinking that could be something we document / reuse in plugins.

```
	func HandlePanicAndFail(logger *log.Logger) {
		if r := recover(); r != nil {
			logger.Panicf("conduit framework experienced a panic: %v", r)
		}
	}
```

Also, it looks like logrus has a "Panic" level. The difference compared to Fatal is that it calls panic after logging instead of Exit.

Reply:
Re-worked, let me know what you think.